tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -435,6 +419,61 @@ test.describe( 'Nested Tabs tests', () => {
 
 		await cleanup( wpAdmin );
 	} );
+
+	test( 'Check if the icons are visible on mobile display on the front end', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await setup( wpAdmin );
+		const editor = await wpAdmin.useElementorCleanPost(),
+			container = await editor.addElement( { elType: 'container' }, 'document' );
+
+		// Add widget.
+		await editor.addWidget( 'nested-tabs', container );
+		await editor.getPreviewFrame().waitForSelector( '.e-n-tabs .e-active' );
+
+		// Act.
+		// Add Icons.
+		await setIconsToTabs( page, TabsIcons );
+
+		// Open front end.
+		await editor.publishAndViewPage();
+		await page.waitForSelector( '.elementor-widget-n-tabs' );
+
+		// Assert
+		await page.setViewportSize( { width: 400, height: 480 } );
```

Comment:
instead of duplicating the dimensions object, I would suggest adding a dimensions object.
This way you'll have one source of truth.

Something like:

```
const viewportSize = {
    desktop: { width: 1920, height: 1080 },
    mobile: { width: 400, height: 480 },
};
```

The implementation should be as follows: 

```
 await page.setViewportSize( viewportSize.mobile );
  await page.setViewportSize( viewportSize.desktop );
```